RELEASE-HOWTO.md

```diff
@@ -15,4 +16,4 @@ Each release will see an increment in the version number acording to [SEMVER](ht
 1. Create a release with the chart as a release artifact. 
 
 ## Post Release 
-1. As the first commit after release update VERSION in Makefile, version in Chart.yaml, heading in CHNAGELOG.md, and tag in values.yaml with the suffix 'next' such that so that further commits are clearly contributions to the next release. Consequently installers from source helm should check out the release tag commit to find the appropriate image by default.
+1. As the first commit after release update VERSION in Makefile, version in Chart.yaml, heading in CHANGELOG.md, and tag in values.yaml to the next odd minor version number to indicate that further commits are clearly contributions to the next release. Consequently installers from source helm should check out the release tag commit to find the appropriate image by default.
```

Comment:
No sure about this odd/even split mechanism. If this step must be odd, then the release process must stipulate even?

Reply:
Mentioned in line 10, can we fine tune post release?